cpptod.dd

```diff
@@ -829,6 +830,86 @@ void test()
 }
 ------
 
+$(HR)
+
+$(H3 $(DDOC_ANCHOR interfaces)Interfaces)
+
+Interfaces in C++ are used to describe the behavior or capabilities of a class without
+commiting to a particular implementation of the class.
+
+$(H4 Interfaces in C++)
+
+  "C++ interfaces are implemented by means of $(I abstract classes). An abstract class
+	defines interface methods as pure virtual functions. A class implementing the interface
+	inherits the abstract class and implements its pure virtual functions."
+
+$(CPPCODE
+class Interface {
+  public:
+    virtual int function() = 0;
+};
+
+class FirstVariantOfImplementation : public Interface {
+  public:
+    int function() {
+      ...
+    }
+}
+
+class SecondVariantOfImplementation : public Interface {
+  public:
+    int function() {
+      ...
+    }
+}
+)
+
+The interface is used as follows:
+
+$(CPPCODE
+FirstVariantOfImplementation FirstVariant;
+SecondVariantOfImplementation SecondVariant;
+
+FirstVariant.function();
+SecondVariant.function();
+)
+
+$(H4 Interfaces in D)
+
+  In D, interfaces (<a href="https://wiki.dlang.org/Books">Alexandrescu 2010, p. 212; Cehreli 2017, p. 347</a>) are implemented
```

Comment:
I'm not very experienced with DDOC but I believe there's a macro to do this rather than inserting the raw anchor directly. Probably better to use that.